src/__test__/components/data-management/ProjectsList.test.jsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { configure, mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import preloadAll from 'jest-next-dynamic';
+import { Provider } from 'react-redux';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import '@testing-library/jest-dom';
+import ProjectsList from '../../../components/data-management/ProjectsList';
+import ProjectCard from '../../../components/data-management/ProjectCard';
+import initialState, { projectTemplate } from '../../../redux/reducers/projects/initialState';
+
+jest.mock('localforage');
+configure({ adapter: new Adapter() });
+const mockStore = configureMockStore([thunk]);
+
+const project1 = {
+  ...projectTemplate,
+  name: 'project 1',
+  uuid: '12345',
+  createdDate: '01-01-2021',
+  lastModified: '01-01-2021',
+};
+
+const project2 = {
+  ...project1,
+  name: 'project 2',
+  uuid: '67890',
+};
+
+const project3 = {
+  ...project1,
+  name: 'testing',
+  uuid: '45678',
+};
+
+const initialStore = mockStore({
+  projects: {
+    ...initialState,
+  },
+});
+
+const filledStore = mockStore({
+  projects: {
+    ...initialState,
+    ids: [project1.uuid, project2.uuid, project3.uuid],
+    meta: {
+      activeProjectUuid: project1.uuid,
+    },
+    [project1.uuid]: project1,
+    [project2.uuid]: project2,
+    [project3.uuid]: project3,
+  },
+});
+
+describe('ProjectsList', () => {
```

Comment:
I don't see any unit tests to test for the functionality of the `matcher` function.

Reply:
It's the test on line 110